libflux/flux-core/src/doc/example.rs

```diff
@@ -132,7 +132,11 @@ fn block_mode(kind: CodeBlockKind) -> Option<BlockMode> {
 //    '# ' - hide the line in the display
 //    '< ' - mark the line as the input
 //    '> ' - mark the line as the output
+//    '#<' - mark the line as the input and hide
+//    '#>' - mark the line as the output and hide
```

Comment:
@sanderson FYI I added this feature to fluxdoc generation.

Reply:
Awesome, do you want to add these here: https://github.com/influxdata/flux/blob/master/docs/fluxdoc.md#example-execution-input-and-output

I can open another PR and add them as well.